subscriptions/management/commands/add_prepend_next_to_subscriptions.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from subscriptions.models import Subscription
+
+
+class Command(BaseCommand):
+    help = ("Active subscription holders need to be informed via audio file "
+            "about the new missed call service.")
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--audio', type=str,
+            help='Audio file containing the notification of the new missed'
+            ' call service.')
+
+    def handle(self, *args, **options):
+        audio_file = options['audio']
+        if not audio_file:
+            raise CommandError('--audio_file is a required parameter')
+        self.stdout.write("Processing active subscriptions ...")
+        count = 0
+        active_subscriptions = Subscription.objects.filter(
+            active=True,
+            messageset__content_type="audio")
+        for active_subscription in active_subscriptions.iterator():
+            # Add audio file to subscription meta_data. Not sure how we'll
```

Comment:
I see the welcome message is currently here: `http://registration.mama.ng.p16n.org/static/audio/registration/eng_NG/welcome_mother.mp3`.
The way I would possibly do this is have a placeholder in the path that the user gives when running the command:
`http://registration.mama.ng.p16n.org/static/audio/registration/<LANG>/welcome_mother.mp3`.
then you can do something like this:
active_subscription.metadata["prepend_next_delivery"] = audio_file.replace('<LANG>', subscription.lang)

Reply:
Am just thinking about this, would I do the formatting of the path or would the user provide this? Am not too clear on that.

Reply:
The user running the command will use this as the audiofile argument: `http://registration.mama.ng.p16n.org/static/audio/registration/<LANG>/welcome_mother.mp3`
Then you can replace `<LANG>` with `subscription.lang`